pkg/asset/installconfig/openstack/session.go

```diff
@@ -23,11 +22,6 @@ type Session struct {
 func GetSession(cloudName string) (*Session, error) {
```

Comment:
I would prefer to see every case where we instantiate an openstack client call this method.